lib/tasks/evm_application.rb

```diff
@@ -110,4 +110,13 @@ def self.update_stop
     File.write(tempfile, " ")
     stop
   end
+
+  def self.set_region_file(region_file, new_region)
```

Comment:
Note, @carbonin, we've leveraging some awesomeness from Pathname to take shortcuts here and in the tests.  I wonder if we should raise an ArgumentError if it's not a Pathname.  Maybe for a followup PR.